templates/kubernetes/auth/auth.yml

```diff
@@ -0,0 +1,87 @@
+apiVersion: oathkeeper.ory.sh/v1alpha1
+kind: Rule
+metadata:
+  name: kratos-public
+spec:
+  upstream:
+    url: http://kratos-public.user-auth
+    stripPath: /.ory/kratos/public
+    preserveHost: true
+  match:
+    #url: http://<backend_service_domain>/.ory/kratos/public/<.*>
+    methods:
+      - GET
+      - POST
+      - PUT
+      - DELETE
+      - PATCH
+  authenticators:
+    - handler: noop
+  authorizer:
+    handler: allow
+  mutators:
+    - handler: noop
+---
+apiVersion: oathkeeper.ory.sh/v1alpha1
+kind: Rule
+metadata:
+  name: kratos-form-data
+spec:
+  upstream:
+    url: http://kratos-admin.user-auth
+    stripPath: /.ory/kratos
+    preserveHost: true
+  match:
+    #url: http://<backend_service_domain>/.ory/kratos/self-service/<(login|registration|recovery|settings)>/flows<.*>
+    methods:
+      - GET
+  authenticators:
+    - handler: noop
+  authorizer:
+    handler: allow
+  mutators:
+    - handler: noop
+---
+apiVersion: oathkeeper.ory.sh/v1alpha1
+kind: Rule
+metadata:
+  name: public-backend-endpoints
+spec:
+  version: test
+  upstream:
+    url: http://<% .Name %>.<% .Name %>
+    preserveHost: true
+  match:
+    # url: http://<backend_service_domain>/status/<.*>
```

Comment:
oathkeeper errors out when more than 1 rule is matched, so I cant really specify a "fallback" route and proxy all